tardis/montecarlo/tests/test_spectrum.py

```diff
@@ -161,9 +161,15 @@ def test_hdf_spectrum(hdf_file_path, spectrum, attr):
     actual = getattr(spectrum, attr)
     if hasattr(actual, "cgs"):
         actual = actual.cgs.value
-    path = os.path.join("spectrum", attr)
-    expected = pd.read_hdf(hdf_file_path, path)
-    assert_almost_equal(actual, expected.values)
+
+    if np.isscalar(actual):
```

Comment:
Since we're saving `delta_frequency` in hdf, this test was failing because it wasn't written in a way to handle scalars - now it's fixed!